datafusion-expr/src/expr.rs

```diff
@@ -696,6 +700,9 @@ fn create_name(e: &Expr, input_schema: &DFSchema) -> Result<String> {
         Expr::Wildcard => Err(DataFusionError::Internal(
             "Create name does not support wildcard".to_string(),
         )),
+        Expr::QualifiedWildcard { .. } => Err(DataFusionError::Internal(
```

Comment:
Maybe we can create a name for `QualifiedWildcard`?

Reply:
I agree including the name of the qualifier in the message might be helpful